.changeset/metal-beds-shout.md

```diff
@@ -0,0 +1,7 @@
+---
+'@segment/analytics-core': patch
+'@segment/analytics-next': patch
+'@segment/analytics-node': patch
```

Comment:
The only package that you need to explicitly bump is core here, all dependent package will be bumped automatically!